### ISSUE TITLE
SIPp 3.5.1 discards 200 OK response

### ISSUE THREAD

**Issue by danu61** [SIPp/sipp#351]
Hi,
My script is expecting a 200 OK with the simple instruction:
```
<recv response="200">
</recv>
```
But upon receiving it, SIPp pops the error message "discarding message which can't be mapped to a known SIPp cal...
Here is the 200 OK message received (taken from the errors.log file). What is wrong with it?
```
SIP/2.0 200 OK
Via: SIP/2.0/UDP 172.16.5.51;branch=z9hG4bK-2066-1-0;rport
From: Line1 <sip:2001@10.47.2.111>;tag=2066SIPpTag00-2
To: Line1 <sip:2001@10.47.2.111>;tag=7c35b18647f0
Call-ID: 1-2066@172.16.5.51@172.16.5.51:5060
CSeq: 1 REGISTER
Contact: <sip:2001@172.16.5.51:5060>;expires=3600
Expires: 3600
Server: NEC SV9300 / V5.3.5
Content-Length: 0
```
Thanks.

**Comment by wdoekes** [comment]
You do realise that I can't really say anything about this without you pasting the scenario and the rest of the messages, right?

Looks like an unexpected callid. And that callid does indeed look strange to me, with two `@` symbols and two IPs.

**Comment by danu61** [comment]
I am so sorry for my incomplete reporting. I was trying to simulate the Call-ID of an actual ONT which looked like this: Call-ID: 5351e060c59b260a@10.47.2.166:5060, so I created a script with the following line Call-ID: [call_id]@[local_ip]:[local_port]. The device I'm testing appears to take that Call-ID and add the IP address again and that caused SIPp to discard the 200 OK. Now I changed the script not to include the IP address and port number in the Call-ID and now SIPp handles the 200 OK that the device replies without issues. As SIPp's error message did not give me any clue which header had the problem I wrote this ticket. I apologize (next time I will include more informiation) and I appreciate your prompt reply and helping me fix my issue.
